### PR TITLE
Update lolbas.py

### PR DESCRIPTION
**Pull request by shannaniggans**
typo

### CODE REVIEW

**Review by sebdraven — APPROVED** [review]

**Comment by tomchop** [comment]
Blocked by https://github.com/yeti-platform/yeti/pull/1056

**Comment by shannaniggans** [comment]
Hahaha, um, i added an "f" to the file. Don't think that should have broken it !

**Comment by tomchop** [comment]
It's not you, just an incompatibility with arango 3.12 that was released ~1 week ago. So we're pinning the versions now, but I just want unittests to pass before merging it :)

**Pull request merged**
